src/js/smart-components/InvitationCreator.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import classnames from 'classnames';
+
+import CopyButton from 'js/components/ui/CopyButton';
+
+import { useUser } from 'js/pages/UserContext';
+
+import { useAuth0 } from 'js/services/react-auth0-spa';
+import { getInvitation } from 'js/services/api';
+
+export default ({ user, className }) => {
+    const { getTokenSilently } = useAuth0();
+    const contextUser = useUser();
+
+    const [urlState, setUrlState] = useState('');
+    const [loadingState, setLoadingState] = useState(false);
+
+    useEffect(() => {
+        setUrlState('');
+        if (!contextUser) {
+            return;
+        };
+
+        const ownedAccount = getOwnedAccount(user);
+        if (!ownedAccount) {
+            return;
+        };
+
+        setLoadingState(true);
+        getInvitation(getTokenSilently, ownedAccount.id)
+            .then(setUrlState)
+            .finally(() => setLoadingState(false));
+    }, [user, getTokenSilently]);
```

Comment:
Adding `getTokenSilently` shouldn't be needed.

```suggestion
    }, [user]);
```

Reply:
_ditto_